app/src/main/java/net/squanchy/injection/ApplicationComponent.java

```diff
@@ -24,17 +28,20 @@
 
     SpeakerRepository speakerRepository();
 
+    ProximityProvider proximityProvider();
+
     class Factory {
 
         private Factory() {
             // non-instantiable
         }
 
-        public static ApplicationComponent create() {
+        public static ApplicationComponent create(Context context) {
             return DaggerApplicationComponent.builder()
                     .firebaseModule(new FirebaseModule())
                     .repositoryModule(new RepositoryModule())
                     .checksumModule(new ChecksumModule())
+                    .proximityModule(new ProximityModule(context))
```

Comment:
I'm a bit confused here: why do we have 2 different modules for the proximity feature (this and the `NearIT*`)?